src/components/v5/common/ActionSidebar/partials/forms/ManagePermissionsForm/ManagePermissionsForm.tsx

```diff
@@ -58,6 +62,15 @@ const ManagePermissionsForm: FC<ActionFormBaseProps> = ({ getFormOptions }) => {
     [togglePermissionsModalOn],
   );
 
+  const ALLOWED_PERMISSION_OPTIONS = PERMISSIONS_OPTIONS.map(
+    ({ options, ...rest }) => ({
+      ...rest,
+      options: options.filter(({ value }) =>
+        value === USER_ROLE.Owner ? Number(team) === Id.RootDomain : true,
+      ),
+    }),
+  );
+
```

Comment:
We need to make this a util, so that we don't repeat code

Reply:
You think so? Both functions are checking the same thing but they go about it differently (I.E. they are filtering different data structures)